extension/src/NABfunctions.ts

```diff
@@ -193,11 +195,21 @@ export async function findNextUnTranslatedText(
         lowerThanTargetState
       );
     }
+    // Run refresh from g.xlf then run again.
+    if (!foundAnything) {
+      await refreshXlfFilesFromGXlf();
+      foundAnything = await LanguageFunctions.findNextUnTranslatedText(
+        settings,
+        SettingsLoader.getAppManifest(),
+        false,
+        languageFunctionsSettings.replaceSelfClosingXlfTags,
+        lowerThanTargetState
+      );
+    }
   } catch (error) {
     showErrorAndLog(error);
     return;
   }
-
   if (!foundAnything) {
     vscode.window.showInformationMessage(
       `No more untranslated texts found. Update XLF files from g.xlf if this was unexpected.`
```

Comment:
This message is not as relevant if we already has done the refresh xlf thingy...

Reply:
Should the whole thing go or just the Update part?

Reply:
Just the update part? I think it's a good thing that you get the info box when there are nothing more to handle